config/initializers/devise_concern_overrides.rb

```diff
@@ -0,0 +1,5 @@
+# Be sure to restart your server when you modify this file.
+
+ActiveSupport.on_load(:action_controller) do
```

Comment:
make sure that devise authy is loading before this and mixing in before this. it can be tricky to get the load right. i thiknk this makes sense too. ooh might be good to make our custom code another method and call it from the rewrite of require_token? so if devise authy ever updates this code [and it breaks] and we need to manually resync, we know what was ours and theirs clearly. [this has happened to me before and it's a BIG pain. we vendored a bunch of js and changed it inline... horrible pain trying to update].

Reply:
cool, yah, that's a good idea 💯